### ISSUE TITLE
Candidate validator page not quite right 

### ISSUE THREAD

**Issue by JonRay15** [vegaprotocol/frontend-monorepo#3683]
## Description

I cant really remember what we agreed here... but "Stake needed for promotion = NULL" doesnt seem like what we wanted! 😂 

## Steps to Reproduce

1. Go to https://governance.vega.xyz/validators
2. Look at the candidate ones 
3. They say stake needed for promotion
4. And its blank

## Expected behavior

I think the answer here might be "the view is designed for standby validators and we dont have any .... and we never designed anything for candidate ones".  So if thats the case we shoudl see it improve when standby validators come in ... but we should still fix it.

## Screenshots

<img width="1045" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/b0989226-3134-47f9-a8a5-9da27351c859">

**Comment by pennyandrews** [comment]
Check this at the end of the day when we have some epoch data and can establish whether this is an issue or not

**Comment by sam-keen** [comment]
@pennyandrews legit problem, still seeing it

**Comment by JonRay15** [comment]
This is because the calc we use for this uses the rewardScore part of the API and that is blank for candidate validators (since they dont earn rewards).

Tecnodes are now (as of this epoch) a standby validator, but since we use rewardScore from the prior epoch we actually need ANOTHER epoch to roll by before we will know if this will sort itself out. 

It will never work for candidates ... but it might be fine for standby ones and since I dont expect us to suddenly acquire another 3 validators in the immediate future that would buy us some time. 

TL;DR ... keep doing nothing and cross your fingers.
